.github/workflows/ci-workflow.yaml

```diff
@@ -0,0 +1,42 @@
+name: CI Workflow
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/lint
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this needed? I don't think so, since it's commented out in the action definition.

Reply:
I need to create a secret in workflow. If I create a secret inside action.yaml the pipeline finished with errors